migrations/upgrades/20191220102232_update_user_created_to_owner.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+
+use Phinx\Migration\AbstractMigration;
+
+class UpdateUserCreatedToOwner extends AbstractMigration
+{
+    public function change()
+    {
+        // To change the type of all the field of user generated collection to owner
+        $this->execute(\Directus\phinx_update(
+            $this->getAdapter(),
+            'directus_fields',
+            [
+                'type' => \Directus\Database\Schema\DataTypes::TYPE_OWNER,
+                'interface' => 'owner'
+            ],
+            ['type' =>  'user_created']
```

Comment:
We should check for the user created interface specifically here. People can be using other interfaces with the user_created type, and we don't want to override those